admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,40 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. You can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired set of nodes.
```

Comment:
To provide some more context on what's going on here wrt node labels (e.g., maybe they didn't know about them during initial installation), maybe at the front of this paragraph start with a line something like:

> Labels can be assigned to node hosts link:../../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation] or link:../../admin_guide/manage_nodes.html#updating-labels-on-nodes[updated post-installation].

Links to those existing sections for your clicking's sake:

https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html#configuring-node-host-labels

https://docs.openshift.com/enterprise/latest/admin_guide/manage_nodes.html#updating-labels-on-nodes